.github/issue_template.md

```diff
@@ -0,0 +1,28 @@
+Hi! Thanks for using JupyterHub.
+
+If you are reporting an issue with JupyterHub:
+
+- Please use the GitHub issue search feature to check if your issue has been
+  asked already. If it has, please add your comments to the existing issue.
+
+- Please fill out the details below to help us troubleshoot the issue that
+  you are facing. Please be as thorough as you are able to provide details on
+  the issue.
```

Comment:
Can we add a 'where applicable' or similar qualifier here? I think some can be intimidated by having several fields that they may not be able to answer; I know I have been. I know the 'as thorough as you are able' covers this, but I'd like to be extra explicit about it.